app/assets/stylesheets/modules/_landing.scss

```diff
@@ -121,3 +121,24 @@
     }
   }
 }
+
+.featured-content {
+  background: $light-gray;
+  border-radius: $base-border-radius;
+  font-size: 12px;
+  margin-bottom: 15px;
+  padding: 10px;
+}
+
+.featured-content-tag {
+  background: shade($base-dark-accent-color, 20%);
+  border-radius: 3px;
+  color: white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.